agent/txn_endpoint_test.go

```diff
@@ -38,10 +40,10 @@ func TestTxnEndpoint_Bad_JSON(t *testing.T) {
 
 func TestTxnEndpoint_Bad_Size_Item(t *testing.T) {
 	t.Parallel()
-	a := NewTestAgent(t, t.Name(), "")
-	defer a.Shutdown()
-
-	buf := bytes.NewBuffer([]byte(fmt.Sprintf(`
+	testIt := func(agent *TestAgent, pass bool) {
```

Comment:
nit: convention is to name expectation args like `wantPass` to make it clearer that it's an assertion not an input.

Reply:
Fixed.